### ISSUE TITLE
DRR renders nothing

### ISSUE THREAD

**Issue by zhuzhemin-sh** [eigenvivek/polypose#20]
I try to run the tutorial: 
https://colab.research.google.com/drive/1ui17wtfjxcjEM9QqMhD4sQXZB2NlMdlS?usp=sharing
from the project page.

But DRR renders nothing in the "Plot the images as a sanity check" cell. I try different combinations including whether to use the inverse of the pose, whether to do "center_volume" but none of them works.

`for inv_pose in [True, False]:
    for center_volume in [True,False]:
        print("inv_pose:",inv_pose,"center_volume:",center_volume)
        # Imaging parameters
        args = dict(
            labels=None,
            orientation="PA",
            height=1436,
            width=1436,
            sdd=1020.0,
            delx=0.194,
            dely=0.194,
            x0=0.0,
            y0=0.0,
            reverse_x_axis=True,
            renderer="trilinear",
            read_kwargs={"bone_attenuation_multiplier": 2.0,"center_volume":center_volume},
            drr_kwargs={"voxel_shift": 0.0},
        )

        # Get the weights of each structure
        weights = torch.load("results/pelvis/weights.pt", weights_only=True)
        
        # Initialize the DRR
        drr = initialize_drr(volume=VOLUME, mask=MASK, **args) # type: ignore
        drr.rescale_detector_(0.125)
        xt = XrayTransforms(drr.detector.height)

        # Load ground truth X-ray images and camera poses
        frames = [10, 9]
        xrays = []
        poses = []
        for idx in frames:
            xray, *_ = read_xray(f"{XRAYS}/{idx:03d}.dcm", crop=100)
            pose, *_ = torch.load(f"{XRAYS}/{idx:03d}.pt")["pose"]
            xrays.append(xray)
            if inv_pose:
                pose_cls = RigidTransform(pose)
                pose_cls_inv = pose_cls.inverse()
                pose = torch.squeeze(pose_cls_inv.matrix,0)
            poses.append(pose)
        gt = xt(torch.concat(xrays)).cuda()
        poses = RigidTransform(torch.stack(poses)).cuda()

        # Plot the images as a sanity check
        img = drr(poses)
        plot(gt, img)
`
<img width="457" height="1778" alt="Image" src="https://github.com/user-attachments/assets/4541c1d1-d68e-4a0b-b6d4-c3b9f55b0047" />

I am also confused that the notebook says "demonstration on the Ljubljana dataset", but the dataset it actually downloads is the "deepfluoro dataset". These are two different dataset on the hugging face. 

The python version I used is Python 3.10.12

[system_gpu_report.txt](https://github.com/user-attachments/files/30696734/system_gpu_report.txt)

[requirements_actual.txt](https://github.com/user-attachments/files/30696603/requirements_actual.txt)

**Comment by eigenvivek** [comment]
Hi @zhuzhemin-sh , thanks for flagging this. The problem is the `xvr-data` repo. I changed the default orientation from PA to AP a month ago, which broke the tutorial. I just pinned the version of the data repo with PA poses (`e4063b8765edb368c03eb3738f29d116ede3147c`). Also, thanks for catching the typo, this was supposed to be a demo on DeepFluoro. Just updated the tutorial notebook and now the plot works:

<img width="348" height="328" alt="Image" src="https://github.com/user-attachments/assets/8124918a-0daf-4765-9aa3-7bd49e47b942" />

**Comment by eigenvivek** [comment]
closing for now, feel free to open a new issue if anything else pops up!

**Comment by zhuzhemin-sh** [comment]
Verified!
